### DISCUSSION TITLE
Frigate+ Image annotation, squirrel or rodents

### DISCUSSION THREAD

**Discussion by RicoKirstein** [blakeblackshear/frigate#15186] [Ask A Question] [answered]
Hi, I bought the Frigate+ Subscription to hopefully track the rodents on our property better. 
I can't receive the basemodel without image annotations and I can't annotate images with the squirrel or rabbit tag or something. 
There seems to be only a small subset of the labels listed on the plus docs available on the annotate website.
Should I just flag everything as a cat and then change it later? is this a bug on the annotation website?
Cheers
Rico

**Comment by NickM-27** [comment, marked as answer]
You need to enable the additional labels in the camera settings in frigate+
